src/components/Feedback/FeedbackCarousel.tsx

```diff
@@ -130,7 +130,9 @@ function FeedbackCarousel(): JSX.Element {
     const path = 'feedback/public/courses';
     const getOnSuccess = (responseData: Data) => {
       // On success, the carousel items will be displayed, allowing multiple feedbacks to be shown.
-      dispatch({ type: 'SET_ITEMS', ...state, items: responseData.data });
+      // the items will be filtered out based on the feedback content.
+      const items = responseData.data.filter((item: Item) => item.attributes.feedback);
+      dispatch({ type: 'SET_ITEMS', ...state, items });
```

Comment:
Awesome arrow filter. It is possible this?
```suggestion
      dispatch({ type: 'SET_ITEMS', ...state, responseData.data.filter((item: Item) => item.attributes.feedback) });
```
or you need the const definition?
Or may be is too long. I have the doubt ...

Reply:
the `items `key  is missing in your suggestion, but yes that could be possible and probably the line should be split to avoid a long line